packages/graphql-transformer-common/src/ResourceConstants.ts

```diff
@@ -29,16 +29,16 @@ export class ResourceConstants {
     OpenSearchStreamingLambdaFunctionLogicalID: 'OpenSearchStreamingLambdaFunction',
     OpenSearchDataSourceLogicalID: 'OpenSearchDataSource',
 
-    // RDS
-    RDSLambdaIAMRoleLogicalID: 'RDSLambdaIAMRole',
-    RDSLambdaLogAccessPolicy: 'RDSLambdaLogAccessPolicy',
-    RDSPatchingLambdaIAMRoleLogicalID: 'RDSPatchingLambdaIAMRole',
-    RDSLambdaLogicalID: 'RDSLambdaLogicalID',
-    RDSPatchingLambdaLogAccessPolicy: 'RDSPatchingLambdaLogAccessPolicy',
-    RDSPatchingLambdaLogicalID: 'RDSPatchingLambdaLogicalID',
-    RDSLambdaDataSourceLogicalID: 'RDSLambdaDataSource',
-    RDSLambdaDataSourceLogicalName: 'RDSLambdaDatabase',
-    RDSPatchingSubscriptionLogicalID: 'RDSPatchingSubscriptionLogicalID',
+    // SQL
+    SQLLambdaIAMRoleLogicalID: 'SQLLambdaIAMRole',
+    SQLLambdaLogAccessPolicy: 'SQLLambdaLogAccessPolicy',
+    SQLPatchingLambdaIAMRoleLogicalID: 'SQLPatchingLambdaIAMRole',
+    SQLLambdaLogicalID: 'SQLLambdaLogicalID',
+    SQLPatchingLambdaLogAccessPolicy: 'SQLPatchingLambdaLogAccessPolicy',
+    SQLPatchingLambdaLogicalID: 'SQLPatchingLambda',
+    SQLLambdaDataSourceLogicalID: 'SQLLambdaDataSource',
+    SQLPatchingSubscriptionLogicalID: 'SQLPatchingSubscription',
```

Comment:
Note that I added `SQLPatchingSubscriptionLogicalID`, which was previously hardcoded in the subscription creation logic